### ISSUE TITLE
viewing module vague error

### ISSUE THREAD

**Issue by rsninja722** [FondationSTaBLFoundation/MOR2Issues#1]
after publishing a module using the publish button, other students can see the module using the generated link

however, when another student tries to view a module with a link generated by using the submit button, they see the following error:

![error](https://media.discordapp.net/attachments/376158910471471104/933217544716845146/Capture.PNG?width=1078&height=604)

this has caused many students to think their links are broken, if possible a message explaining the view does not have permission instead of an error message would remove a lot of confusion

**Comment by CSchank** [comment]
Yes, only TAs have permission to view links from the submit button. But I agree it could be confusing. This is a great suggestion, thank you! Btw, this happens because IHP's default is to show an error when access is denied. I believe we can put custom error views though. We'll look into that for sure.
